Fix #4773: Add responsive behavior to the landing page

nithusha21 opened the pull request:
## Explanation
Fixes #4773 This PR adds responsive behavior to the landing page

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes.
- [x] The PR explanation includes the words "Fixes #bugnum: ...".
- [x] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python scripts/pre_commit_linter.py` and `bash scripts/run_frontend_tests.sh`.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [x] The PR is assigned to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue.

nithusha21 committed:
added new fractions landing pages and removed old landing pages

nithusha21 committed:
Fixed linting errors

nithusha21 committed:
made review changes

nithusha21 committed:
linting fixes

nithusha21 committed:
linting fixes

nithusha21 committed:
Some changes

nithusha21 committed:
made the landing page responsive

nithusha21 committed:
linting fixes

nithusha21 committed:
Merge branch 'develop' into landing-page-fix

nithusha21 committed:
changed image for mobile display on teacher page

seanlip commented on core/templates/dev/head/pages/landing/fractions/Fractions.js:
```diff
@@ -22,16 +22,24 @@ oppia.controller('Fractions', [
       siteAnalyticsService.registerOpenFractionsFromLandingPageEvent(
         viewerType);
       $timeout(function() {
         $window.location = ' /collection/4UgTQUc1tala';
```
Why is there a leading space in the string?

nithusha21 replied:
Oops fixed that.

seanlip commented on core/templates/dev/head/pages/landing/fractions/Fractions.js:
```diff
@@ -22,16 +22,24 @@ oppia.controller('Fractions', [
+
+    $scope.onClickLearnMoreButton = function() {
+      $timeout(function() {
+        $window.location = 'https://www.oppia.org/splash';
```
`/splash`, otherwise it'll go to the wrong place when you're testing on the test server.

nithusha21 replied:
Okay. fixed that

seanlip commented on core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html:
```diff
@@ -7,243 +7,282 @@
+            </div>
+            <div class="col-sm-6">
+              <h2 class="oppia-fractions-landing-h2" style="color: #242424; text-align: right; right: 10%">With Oppia, we make it easy to go through the lessons and learn something new!
+                <br><br>You will work to learn new concepts and help Matt become a great baker along the way. We make it fun and easy to get started.
```
Just realized something is off about the text on this page. Suggest:

You will work ... --> Students will work ...
Matt --> Matthew

(edit: please check the mocks again, this text is not correct)

nithusha21 replied:
I had missed this when I finally pasted the student page to the teacher page. Fixed it now. Sorry about that

seanlip commented:
Also, please delete any image asset files that aren't used.

seanlip commented:
Bug report: on a larger screen, images seem to be getting cut off. See the arrow in the image below.

![test](https://user-images.githubusercontent.com/10575562/37303825-e4fc910e-25ec-11e8-898b-e0cae06e9a8a.png)

Also if you zoom out farther, the main content detaches from the side of the screen:

![zoomout2](https://user-images.githubusercontent.com/10575562/37303883-263fe65c-25ed-11e8-820f-833371bd57ef.png)

nithusha21 committed:
made review changes and bug fixes

nithusha21 commented:
Deleted all the unused assets and also fixed the large screen styling.  In my opinion the page looks nice upto laptop view and doesn't break at higher sizes but doesn't look as appealing 😅. Maybe it's just because I am viewing it at 50% zoom.

seanlip commented on core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html:
```diff
@@ -7,243 +7,281 @@
+            </div>
+            <div class="col-sm-6">
+              <h2 class="oppia-fractions-landing-h2" style="color: #242424; text-align: right; right: 80px">With Oppia, we make it easy to go through the lessons and learn something new!
+                <br><br>Students get to work through stories and apply their knowledge of real-world problems.
```
Let's change this to "knowledge to".

nithusha21 replied:
Changed this

seanlip reviewed (approved):
Hi @nithusha21 -- thanks, looks good! One very small wording change and it's good to go.

nithusha21 committed:
changed of -> to

seanlip commented:
Thanks, LGTM!

seanlip commented:
@BenHenning, FYI -- this needs cherrypicking!
